docs/tutorial/VueJS/step9.md

```diff
@@ -90,6 +90,14 @@ So, let's create a new Javascript file `lib/tutorial.js` to handle everything re
 	<!-- After all script loads -->
 	<script src="js/components/SugarTutorial.js"></script>
 ```
+```js
+// Register the component
+const app = Vue.createApp({
+	components: {
+		"sugar-tutorial": SugarTutorial,
+		...
+	},
+```
 
 ## Localize the tutorial
 
```

Comment:
Changing this to

locales/`<lang>`.json

`<lang>`.json

```
	"TutoPrev": "Prev",
	"TutoNext": "Next",
	"TutoExplainTitle": "Pawn Activity",
	"TutoExplainContent": "Welcome into the Pawn activity. This activity is an activity to test Sugarizer development.",
	"TutoAddTitle": "Add pawn",
	"TutoAddContent": "Click here to add one to three pawns on the board.",
	"TutoBackgroundTitle": "Change background",
	"TutoBackgroundContent": "Click here to choose a new background for the board.",
```

Reply:
done, thanks!